scripts/spell-check.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+#
+## Copyright (C) 2020 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+#
+# This script runs codespell against selected files.
+#
+# Use -q or --quiet option to run quietly
+#
+
```

Comment:
Have you considered something like `set -e` to avoid cascading or undetected errors causing problems, especially if this script is used in a "make" target or another script.

Reply:
Added "set -e" in next commit.